test_fms/mpp/test_read_input_nml.F90

```diff
@@ -68,7 +68,7 @@ program test_read_input_nml
   n = 1
   do
     read(1, '(A)', iostat=stat) line
-    if (stat.eq.-1) then
+    if (stat < 0) then
```

Comment:
You should use the _is_iostat_end_ function that came in with the 2003 standard for testing the status code.